include/swift/Runtime/Metadata.h

```diff
@@ -3061,16 +3065,9 @@ class TargetClassDescriptor final
   using TrailingGenericContextObjects::getGenericContextHeader;
   using TrailingGenericContextObjects::getFullGenericContextHeader;
 
-  /// This bit is set in the context descriptor header's kind-specific flags
-  /// if this is a class descriptor with a vtable descriptor for runtime
-  /// vtable instantiation.
-  static constexpr const uint16_t HasVTableFlag =
-    uint16_t(TypeContextDescriptorFlags::HasVTable);
-  /// This bit is set in the context descriptor header's kind-specific flags
-  /// if this is a class descriptor with a resilient superclass.
-  static constexpr const uint16_t HasResilientSuperclassFlag =
-    uint16_t(TypeContextDescriptorFlags::HasResilientSuperclass);
-  
+  /// The superclass of this class.  To interpret this pointer, 
```

Comment:
Comment's cut off here.

Reply:
Oops.  I'll fix in this PR if I have to submit again, otherwise it'll go in the next one.